src/contrib/mlir/dialect/dialect.hpp

```diff
@@ -25,15 +25,17 @@
 #include "ngraph/check.hpp"
 namespace mlir
 {
-    class NGDialect : public mlir::Dialect
+    class NGraphOpsDialect : public mlir::Dialect
```

Comment:
Is the rename needed ?

Reply:
Just for alignment with MLIR changes. At the beginning I added the whole dialect as legal in the lowering. Then I saw they had renamed affine/standard dialects in the same way (StandardOpsDialect, AffineOpsDialect) and try to align ours with that.